src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets

```diff
@@ -115,4 +115,12 @@ Copyright (c) .NET Foundation. All rights reserved.
       Include="$(MSBuildThisFileDirectory)..\codestyle\vb\Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes.dll"
       IsImplicitlyDefined="true" />
   </ItemGroup>
+
+  <!-- ILLinker Analyzers -->
+  <ItemGroup Condition="'$(PublishSingleFile)' == 'true' Or
+                        '$(PublishTrimmed)' == 'true'">
+    <Analyzer
```

Comment:
There is no user-facing way to disable this analyzer being included. Consider adding some mechanism to do this (for example, you may want to disable them for performance related reasons).

here is an example of a way to approach this: https://github.com/dotnet/sdk/blob/71ab1914a709bbeae008535d1e694bb8e81ab707/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets#L73-L85

Reply:
Yeah we should probably add `EnableNETAnalyzers` to the condition